tests/unit/test_split.py

```diff
@@ -309,7 +305,6 @@ def test_non_xml_method():
 def test_linked_server():
     seed = """
     SELECT *
-    INTO linked_server.db.schm.table1
```

Comment:
Based on the description of the PR as well as the commit messages it's not obvious to me why we would no longer be able to support this.